### ISSUE TITLE
PyPI download does not correctly include `attrs` as a dependency

### ISSUE THREAD

**Issue by forana** [Doist/todoist-api-python#26]
## Bug description
`attrs` is not included when following install instructions. In a fresh virtualenv:

```
$ pip install todoist-api-python==1.1.0
Looking in indexes: https://pypi.org/simple, https://www.piwheels.org/simple
Collecting todoist-api-python==1.1.0
  Using cached https://www.piwheels.org/simple/todoist-api-python/todoist_api_python-1.1.0-py3-none-any.whl (11 kB)
Collecting requests<3.0.0,>=2.26.0
  Using cached https://www.piwheels.org/simple/requests/requests-2.27.1-py2.py3-none-any.whl (63 kB)
Collecting certifi>=2017.4.17
  Using cached https://www.piwheels.org/simple/certifi/certifi-2021.10.8-py2.py3-none-any.whl (151 kB)
Collecting urllib3<1.27,>=1.21.1
  Using cached https://www.piwheels.org/simple/urllib3/urllib3-1.26.8-py2.py3-none-any.whl (155 kB)
Collecting idna<4,>=2.5
  Using cached https://www.piwheels.org/simple/idna/idna-3.3-py3-none-any.whl (64 kB)
Collecting charset-normalizer~=2.0.0
  Using cached https://www.piwheels.org/simple/charset-normalizer/charset_normalizer-2.0.11-py3-none-any.whl (44 kB)
Installing collected packages: urllib3, idna, charset-normalizer, certifi, requests, todoist-api-python
Successfully installed certifi-2021.10.8 charset-normalizer-2.0.11 idna-3.3 requests-2.27.1 todoist-api-python-1.1.0 urllib3-1.26.8
```

pip freeze:

```
$ pip freeze
certifi==2021.10.8
charset-normalizer==2.0.11
idna==3.3
requests==2.27.1
todoist-api-python==1.1.0
urllib3==1.26.8
```

## Expected behaviour
Documentation instructions lead to success.

## Is reproducible
Yes

## Version information:
- Package version: v1.1.0
- Python version: 3.9

## Additional information
It looks like #10 fixed this, but has not been released.

**Comment by leinadao** [comment]
Also ran into this issue with a fresh project and poetry install.

**Comment by lefcha** [comment]
We just released [v1.1.1](https://pypi.org/project/todoist-api-python/1.1.1/).

**Comment by leinadao** [comment]
Tested now working for me in poetry, thanks.
Removed manual attrs, upgraded todoist-api-python, removed venv and re-installed.
